lib/questions.rb

```diff
@@ -63,28 +73,34 @@ def separate_array_into_even_and_odd_numbers(array)
 # e.g. 'bob'. So in the array ['bob', 'radar', 'eat'], there
 # are 2 palindromes (bob and radar), so the method should return 2
 def number_of_elements_that_are_palindromes(array)
+  array.select{|e| e.reverse == e}.length
 end
 
 # return the shortest word in an array
 def shortest_word_in_array(array)
+  array.min {|a,b| a.length <=> b.length}
```

Comment:
Space missing after comma.<br>Space between { and | missing.<br>Space missing inside }.